tests/Functional/Controller/CoreControllerTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Tests\Functional\Controller;
+
+use PHPUnit\Framework\TestCase;
+use Sonata\AdminBundle\Tests\Functional\AppKernel;
+use Symfony\Bundle\FrameworkBundle\Client;
+
+class CoreControllerTest extends TestCase
```

Comment:
Just to increase the code coverage measurement for `DashboardAction`, I'd call this class `DashboardActionTest`.

Reply:
Done